tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV1Rules.cs

```diff
@@ -657,6 +657,17 @@ public static Result IsGrpcServerV1(this MockSpan span, ISet<string> excludeTags
                 .Matches("component", "Grpc")
                 .Matches("span.kind", "server"));
 
+        public static Result IsHangfireV1(this MockSpan span) => Result.FromSpan(span)
+           .Properties(
+                s => s
+                    .MatchesOneOf(Name,  "Hangfire.onPerform")
+                    .Matches(Type, "Hangfire"))
+           .Tags(s => s
```

Comment:
nit: Can you re-use the same formatting as the other methods, both here and the SpanMetadataV0Rules.cs file?